cli/types/cypress.d.ts

```diff
@@ -3117,6 +3122,11 @@ declare namespace Cypress {
     onAnyAbort(route: RouteOptions, proxy: any): void
   }
 
+  interface Session {
+    clearAllSavedSessions: () => Promise<void>;
```

Comment:
Mind adding a comment to describe clearAllSavedSessions? It helps with maintenance. You can pull something from https://docs.cypress.io/api/cypress-api/session.

Reply:
Unfortunately there's no docs there either, but I can try to write something up.

Reply:
It mentions something like;` Clear all saved sessions and re-run the current spec file.`

Reply:
Yeah sorry, I missed that at first.  :shame:

Reply:
No worries! It's not super obvious